local_sales/app/views/customers/_category_card.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="group relative">
+  <div class="w-full rounded-md group-hover:opacity-75 lg:h-100 lg:aspect-none">
+    <%= image_tag('category', class: 'w-full h-full object-center object-cover lg:w-full lg:h-full') %>
```

Comment:
As a suggestion add the alt attribute to the image_tag
example:
`<% = image_tag ("image_path", class: "image styles", alt: "Image description")%>`